bin/OSPData/adera/ph_rcs.sturdy.gltf

```diff
@@ -8,49 +8,33 @@
         {
             "name" : "Scene",
             "nodes" : [
-                3
+                2
             ]
         }
     ],
     "nodes" : [
         {
             "name" : "fx_plume_phRCSplume",
-            "rotation" : [
-                0,
-                1,
-                0,
-                -1.6292068494294654e-07
-            ],
             "translation" : [
                 0,
                 0,
-                0.1351655125617981
+                -0.1351655125617981
```

Comment:
These are awfully arbitrary.

Do we need named-constant support in the gltf files?

Reply:
These aren't modified manually, they're exported from Blender and are bound to have ugly numbers. I do often try to model things with nice numbers but ultimately there are bound to be tons of arbitrary values so don't worry about it in these.

Reply:
Ah, ok.